pkg/gencmd/main.go

```diff
@@ -0,0 +1,136 @@
+package gencmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type Cmd interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Cmd should have comment or be unexported